header.php

```diff
@@ -57,7 +57,7 @@
 			<nav id="site-navigation" class="main-navigation" role="navigation">
 				<h4 class="menu-toggle"></h4>
 
-				<?php wp_nav_menu( array( 'theme_location' => 'primary', 'container_class' => 'menu-primary-container' ) ); ?>
+				<?php wp_nav_menu( array( 'theme_location' => 'primary', 'menu_class' => 'clearfix ', 'container_class' => 'menu-primary-container' ) ); ?>
```

Comment:
https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#indentation

**For associative arrays, each item should start on a new line when the array contains more than one item:**